source/frontend/src/features/documents/ComposedDocument.ts

```diff
@@ -16,7 +16,7 @@ import Api_TypeCode from 'models/api/TypeCode';
 
 export interface ComposedDocument {
   mayanMetadata?: Api_Storage_DocumentMetadata[];
-  pimsDocument?: Api_DocumentRelationship;
+  pimsDocumentRelationship?: Api_DocumentRelationship;
```

Comment:
get the feeling I have seen this code before :)

Reply:
Yeah, I've struggled with refactoring this code without unit tests or enforced typings.